pull request for TSE review

muhenge opened the pull request.

muhenge committed:
Chore-Added all project files: linter, bootstrap file, and html, css

muhenge committed:
Feature: header

muhenge committed:
Feature: I added a column-left

muhenge committed:
Bug: fixed css path in html

muhenge committed:
Chore: added bootstrap cdn for online preview

muhenge committed:
bg-restarting

muhenge committed:
ft-I added all section form the review

muhenge committed:
I added some heaser but not yet done

muhenge committed:
I added some pictures and

muhenge committed:
Feature- I added the header and the responsiveness

muhenge committed:
Feature-I added the header and the navbar

muhenge committed:
bug-fixing stylelint

muhenge committed:
Bug-fixing lint for css

muhenge committed:
Bug-fixes css lint

muhenge committed:
Bug- corrected lint errors and update the css

muhenge committed:
added position to badges

muhenge committed:
Updating the readme.md

muhenge committed:
Changes in the file Readme

muhenge committed:
Update README.md

suretrust commented on index.html:
```diff
@@ -0,0 +1,1704 @@
+                        aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon text-white">...</span>
+
+                    </button>
```
##### Navigation bar
* [x] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/)). Your navbar is not properly sized and contains sub navbar on the medium screen.

**Yours**
<img width="854" alt="Screenshot 2020-01-03 at 17 12 15" src="https://user-images.githubusercontent.com/31304943/71734404-359b8c00-2e4c-11ea-80d8-f8f21275dcb1.png">


**Original**
<img width="981" alt="Screenshot 2020-01-03 at 17 11 27" src="https://user-images.githubusercontent.com/31304943/71734384-20266200-2e4c-11ea-9958-304fd9de994d.png">

suretrust commented on index.html:
```diff
@@ -0,0 +1,1704 @@
+                    <div class="col-12 col-md-9">
+                        <div class="divider w-100 d-flex justify-content-center">
+                            <div class="newsweek-logo-s"></div>
+                            <span>IN THE MAGAZINE</span>
+                        </div>
+                        <div class="row">
+                            <div class="col-12 col-md-4">
+                                <article class="magazine-stories">
+                                    <a href="#"><img class="img-magazine"
+                                            src="https://d.newsweek.com/en/full/1548831/fe-csr-01-1041330598-banner.webp?w=466&h=311&l=52&t=49&q=90&f=e6d8a68693cdcf8906bbbe92a804bc55"
```
The "IN THE MAGAZINE" section is messy on a medium screen. [Link to page](https://www.newsweek.com/).

**Yours**
<img width="854" alt="Screenshot 2020-01-03 at 17 15 48" src="https://user-images.githubusercontent.com/31304943/71734755-e86bea00-2e4c-11ea-8bd7-2b68d553c99d.png">

**Original**
<img width="979" alt="Screenshot 2020-01-03 at 17 16 06" src="https://user-images.githubusercontent.com/31304943/71734775-f28de880-2e4c-11ea-87db-08a1f7901f77.png">

suretrust commented on index.html:
```diff
@@ -0,0 +1,1704 @@
+                                            <a class="links font-weight-bold" href="#">
+                                                Lorem ipsum dolor sit</a>
+                                        </h2>
+                       
```
##### Footer

* [ ] It contains social media icons ([link to page](https://www.newsweek.com/))
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

suretrust reviewed (changes requested):
Hi @muhenge,

### Great job thus far. 🙌 Please see my comments on your code.

**Kindly submit for another code review once these issues have been resolved.** _Happy coding!_ 🎉

muhenge committed:
Bug- corrected the navbar and medium sized section

I corrected the navbar and the subnav on the medium screen and corrected responsiveness inside one of the section in the medium screen size

muhenge committed:
Added the footer

this footer is not yet done properly

muhenge committed:
Bug- I fixed the responsiveness and added social icons

izaiasneto4 commented:
#### Hi @muhenge. Great job implement all of the required changes. Status: Approved :+1: 

TSE: Izaias Neto

izaiasneto4 commented:
### Hi @muhenge. Nice job implementing the requested changes, there's just one thing left to fix before we can get your project approved.  :arrow_down: 

- [ ] Let's fix all of the errors given by the [validator](https://validator.w3.org/nu/#file)

Please, when finished submit the project for a new request. :pray:

muhenge committed:
Bug- fixing all W3C validation errors

mosesogwo reviewed (approved):
### :trophy: Congratulations! You have done well.

Please consider aligning the three columns at the top properly before merging to master. Since this wasn't pointed out before now, I'll give my approval.

![image](https://user-images.githubusercontent.com/46611870/72217997-fbaa4400-3535-11ea-933b-d0d4a6edcba7.png)


Cheers.

### Status: Approved
